ngraph/src/ngraph/op/split.cpp

```diff
@@ -186,3 +189,65 @@ shared_ptr<Node> op::v1::Split::clone_with_new_inputs(const OutputVector& new_ar
     check_new_args_count(this, new_args);
     return make_shared<v1::Split>(new_args.at(0), new_args.at(1), m_num_splits);
 }
+
+namespace
+{
+    inline bool evaluate(const HostTensorPtr& in,
+                         const HostTensorPtr& out,
+                         const Coordinate& lower_bounds,
+                         const Coordinate& upper_bounds)
+    {
+        runtime::reference::slice(in->get_data_ptr<const char>(),
+                                  out->get_data_ptr<char>(),
+                                  in->get_shape(),
+                                  lower_bounds,
+                                  upper_bounds,
+                                  Strides(lower_bounds.size(), 1),
+                                  out->get_shape(),
+                                  in->get_element_type());
+
+        return true;
+    }
+
+    bool evaluate_split(const HostTensorPtr& data_tensor,
+                        const HostTensorPtr& axis_tensor,
+                        const HostTensorVector& outputs,
+                        const int64_t num_splits,
+                        const Node* split_node)
+    {
+        bool rc = true;
```

Comment:
Why do we need this variable?
As I can see we don't use it in this code.

Reply:
Since template attitude is not used it is not needed, removed